docs/APIExamples.rst

```diff
@@ -0,0 +1,570 @@
+=================
+API Examples
+=================
+
```

Comment:
Please give a brief introduction at the top of the page.  Just a paragraph or two about the API and how it can help researchers or the generally curious.  It might be good to include Apple/Linux/Windows specific notes here too (if any).  Also that we're compliant with python 3.5+.